oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCUserManager.java

```diff
@@ -204,7 +204,10 @@ private int sendLogout()
         try {
             logoutURI = this.configuration.getLogoutOIDCEndpoint();
         } catch (Exception e) {
-            this.logger.error("No OIDC logout URI set", e);
+            /**
```

Comment:
I was not suggesting you a comment to put in the code, I was telling you that the error message you have in the log does not make any sense since not having a logout URI is not going to cause an exception.

Reply:
No exception = No handling. Should I close the JIRA issue and this PR?

Reply:
I don't understand your question. My only point here and in all my previous comments is that the message you have in the log is not accurate, so you should fix it (and remove the new comment you added which does not make sense).

Reply:
Okay, doing that. ✅

Reply:
It seems we don't understand each other: I did not suggest you to stop logging an error, I just indicated that the error message you chose was not accurate and that it should be a different one.

Reply:
Oh okay, I understood it incorrectly. 
We have to log the appropriate errors in both the catch blocks.
```
try {
            logoutURI = this.configuration.getLogoutOIDCEndpoint();
        } catch (Exception e) {
            // TODO: log something?
        }

        if (logoutURI != null) {
            try {
                ret = sendLogout(logoutURI, this.configuration.getIdToken());
            } catch (Exception e) {
                // TODO: log something?
            }
        } else {
            this.logger.debug("Don't send OIDC logout request: no OIDC logout URI set");
        }